package.json

```diff
@@ -2,18 +2,136 @@
+    "name": "Weston Ruter",
+    "email": "weston.ruter@xwp.co",
+    "url": "https://www.xwp.co"
+  },
+  "contributors": [
+    {
+      "name": "Piotr Delawski",
+      "email": "piotr.delawski@xwp.co"
+    },
+    {
+      "name": "Mike Crantea",
+      "email": "mike.crantea@xwp.co"
+    },
+    {
+      "name": "Justin Kopepasah",
+      "email": "justin.kopepasah@xwp.co",
+      "url": "https://kopepasah.com"
+    }
+  ],
   "license": "MIT",
-  "homepage": "https://github.com/xwp/wp-dev-lib#readme"
+  "homepage": "https://github.com/xwp/wp-dev-lib#readme",
+  "scripts": {
+    "build:self": "./build-workflows.sh",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "browserslist": [
+    "last 2 Chrome versions",
+    "last 2 Firefox versions",
+    "last 2 Safari versions",
+    "last 2 Edge versions",
+    "last 2 Opera versions",
+    "last 2 iOS versions",
+    "last 1 Android version",
+    "last 1 ChromeAndroid version",
+    "ie 11",
+    "> 1%"
+  ],
+  "babel": {
+    "presets": [
+      [
+        "env"
+      ]
+    ]
+  },
+  "stylelint": {
+    "extends": "stylelint-config-wordpress/scss",
+    "plugins": [
+      "stylelint-no-unsupported-browser-features"
+    ],
+    "rules": {
+      "string-quotes": "single",
+      "color-hex-length": null,
+      "function-parentheses-space-inside": null,
+      "function-url-quotes": "always",
+      "at-rule-empty-line-before": [
+        "always",
+        {
+          "except": [
+            "first-nested",
+            "blockless-after-same-name-blockless"
+          ],
+          "ignore": [
+            "after-comment"
+          ],
+          "ignoreAtRules": [
+            "else",
+            "elseif"
+          ]
+        }
+      ],
+      "plugin/no-unsupported-browser-features": [
+        true,
+        {
+          "severity": "warning"
+        }
+      ],
+      "selector-pseudo-element-colon-notation": "double",
+      "value-keyword-case": null
+    }
+  },
+  "dependencies": {
+    "autoprefixer": "^7.1.2",
+    "babel-cli": "^6.24.1",
+    "babel-core": "^6.25.0",
+    "babel-loader": "^7.1.1",
+    "babel-preset-env": "^1.6.0",
+    "del": "^3.0.0",
+    "eslint": "^4.3.0",
+    "eslint-config-wordpress": "^2.0.0",
+    "eslint-loader": "^1.9.0",
+    "eslint-plugin-compat": "^1.0.4",
+    "gulp": "gulpjs/gulp.git#4.0",
+    "gulp-cached": "^1.1.1",
+    "gulp-eslint": "^4.0.0",
+    "gulp-if": "^2.0.2",
+    "gulp-imagemin": "^3.3.0",
+    "gulp-plumber": "^1.1.0",
+    "gulp-postcss": "^7.0.0",
+    "gulp-progeny": "^0.4.0",
+    "gulp-sass": "^3.1.0",
+    "gulp-sourcemaps": "^2.6.0",
+    "gulp-watch": "^4.3.11",
+    "lodash": "^4.17.4",
+    "postcss": "^6.0.8",
+    "postcss-assets": "^4.2.0",
+    "postcss-cssnext": "^3.0.2",
+    "postcss-pxtorem": "^4.0.1",
+    "postcss-reporter": "^4.0.0",
+    "postcss-scss": "^1.0.2",
+    "progress-bar-webpack-plugin": "^1.10.0",
+    "require-dir": "^0.3.2",
+    "stylelint": "^7.13.0",
```

Comment:
This should be `^8.0.0` as `stylelint-config-wordpress`'s `12.0.0` requires stylelint `8.0.0`